connect.go

```diff
@@ -213,12 +213,6 @@ func initArdopTNC() error {
 		return fmt.Errorf("ARDOP TNC initialization failed: %w", err)
 	}
 
-	if !config.Ardop.ARQBandwidth.IsZero() {
```

Comment:
With the recent changes, shouldn't this be reverted? 🤔

Reply:
Rolled that back.